src/Glpi/Form/AccessControl/FormAccessControl.php

```diff
@@ -264,6 +265,48 @@ public function createConfigFromUserInput(array $input): JsonConfigInterface
         return $strategy->createConfigFromUserInput($input);
     }
 
+    /**
+     * Encode the input name to make sure it is unique and multiple items
+     * can be updated using a single form.
+     *
+     * Content must be decoded using `splitEncodedInputs`.
+     *
+     * @param string $name
+     * @return string
+     */
+    public function encodeInputName(string $name): string
+    {
+        return "_access_control_{$this->getID()}_$name";
```

Comment:
Why not using something like `_access_control[{$this->getID()}][$name]` ? It could remove the requirement of the `splitEncodedInputs` logic.